From 0615b213cb2b87aff50aec654bfa5994b6fe6632 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 30 Apr 2019 16:48:07 +0300
Subject: [PATCH] [SW-0] include: specify non local includes as such

The includes which are not located in the local directory, but
provided by the build system should be surrounded by triangular
brackets.
Use the more correct way.

Change-Id: If1b780881add4a6cf9c46c3d8311d261358fec0e
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/gaudi.c b/gaudi/gaudi.c
index 08e89a1..3fa484a 100644
--- a/gaudi/gaudi.c
+++ b/gaudi/gaudi.c
@@ -8,9 +8,10 @@
 #include <psoc_global_conf_regs.h>
 #include <pcie_aux_regs.h>
 
-#include "preboot.h"
-#include "pll.h"
-#include "uart.h"
+#include <preboot.h>
+#include <pll.h>
+#include <uart.h>
+
 #include "gaudi.h"
 
 void preboot_pcie_wait_for_link(void)
diff --git a/goya/goya.c b/goya/goya.c
index f9a8838..748f81e 100644
--- a/goya/goya.c
+++ b/goya/goya.c
@@ -12,8 +12,9 @@
 #include <cpu_timestamp_regs.h>
 #include <pcie_db_rsv_regs.h>
 
-#include "preboot.h"
-#include "pll.h"
+#include <preboot.h>
+#include <pll.h>
+
 #include "goya.h"
 
 #define BTL_COUNTER_MAX		BIT(22)
